src/app/components/Figure/Caption/index.jsx

```diff
@@ -1,11 +1,24 @@
 import styled from 'styled-components';
 import { FF_NEWS_SANS_REG, C_STORM } from '../../../lib/constants/styles';
+import mediaQuery from '../../../helpers/mediaQueries';
 
 const Caption = styled.figcaption`
   background-color: #d5d0cd;
   color: ${C_STORM};
   font-family: ${FF_NEWS_SANS_REG};
+  font-size: 0.875em;
```

Comment:
@benjaminhobbs have you thought about using font size and line height variables like we are doing for colours? e.g. font-size: ${canon}; https://github.com/bbc/gel-typography/blob/master/lib/_settings.scss#L76

Reply:
Yes and it shouldn't be done with styled components. Also there's only one use of each so factoring it out would be wrong in general.

Reply:
Interesting, thanks for the reply